src/pages/RFOX/components/RFOXHeader.tsx

```diff
@@ -25,14 +25,15 @@ export const RFOXHeader = () => {
     selectPortfolioAccountIdsByAssetIdFilter(state, accountIdsFilter),
   )
 
-  const activeAccountDropdown = useMemo(() => {
-    // Side-effect is useMemo isn't a mistake - reactivity is borked if trying to do an `useEffect` instead, see
-    // https://github.com/shapeshift/web/pull/7509#issuecomment-2297664261
+  useEffect(() => {
     if (accountIds.length === 1) {
       setStakingAssetAccountId(accountIds[0])
-      return null
     }
-    if (!(accountIds.length > 1)) return null
```

Comment:
No weird side effects anymore 👋🏽 this is now handled gracefully in a `useEffect` as it should now that reactivity is happy